products/sentinel_one.py

```diff
@@ -502,7 +505,8 @@ def process_search(self, tag: Tag, base_query: dict, query: str) -> None:
     def parameter_mapping(self) -> dict[str, list[str]]:
         return PARAMETER_MAPPING_PQ if self._pq else PARAMETER_MAPPING_DV
 
-    def nested_process_search(self, tag: Tag, criteria: dict, base_query: dict) -> None:
+    def nested_process_search(self, tag: Tag, criteria: dict, base_query: dict, json_output: bool) -> None:
+        self._json = json_output
```

Comment:
For consistent output for the duration of using this product file, `self._json` should be set in the `__init__` function and then only referenced in other parts of the code

```suggestion
    def nested_process_search(self, tag: Tag, criteria: dict, base_query: dict) -> None:
```